pom.xml

```diff
@@ -1377,6 +1383,10 @@
             <name>OSSRH Public Releases Group</name>
             <url>https://oss.sonatype.org/content/repositories/releases/</url>
         </repository>
+        <repository>
```

Comment:
Do we need this ?

Reply:
We don't need it actually. I have removed it in the updated changes